x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -48,11 +46,11 @@ export const META_ENGINE_SOURCE_ENGINES_PATH = `${ENGINE_PATH}/engines`;
 
 export const ENGINE_RELEVANCE_TUNING_PATH = `${ENGINE_PATH}/relevance_tuning`;
 export const ENGINE_SYNONYMS_PATH = `${ENGINE_PATH}/synonyms`;
-export const ENGINE_RESULT_SETTINGS_PATH = `${ENGINE_PATH}/result-settings`;
+export const ENGINE_RESULT_SETTINGS_PATH = `${ENGINE_PATH}/result_settings`;
 
 export const ENGINE_CURATIONS_PATH = `${ENGINE_PATH}/curations`;
 export const ENGINE_CURATIONS_NEW_PATH = `${ENGINE_CURATIONS_PATH}/new`;
 export const ENGINE_CURATION_PATH = `${ENGINE_CURATIONS_PATH}/:curationId`;
 
-export const ENGINE_SEARCH_UI_PATH = `${ENGINE_PATH}/reference_application/new`;
+export const ENGINE_SEARCH_UI_PATH = `${ENGINE_PATH}/search_ui`;
```

Comment:
@JasonStoltz feel free to yell if you strongly prefer keeping the `/new` - I figure the `/search_ui` change is pretty straightforward but was wasn't sure about the usefulness of the `/new` pathing 🤔